src/external/fees/FeeManager_v1.sol

```diff
@@ -73,7 +73,7 @@ contract FeeManager_v1 is ERC165, IFeeManager_v1, Ownable2StepUpgradeable {
     //--------------------------------------------------------------------------
     // Initialization
 
-    function init( //@note instead constructor?
+    function init(
```

Comment:
Is your question basically whether this contract should be upgradeable or not?

Reply:
Yeah basically